pkg/joinserver/grpc_application_activation_settings_registry.go

```diff
@@ -76,22 +82,8 @@ func (srv applicationActivationSettingsRegistryServer) Set(ctx context.Context,
 		)
 	}
 	v, err := srv.JS.applicationActivationSettings.SetByID(ctx, req.ApplicationIdentifiers, req.FieldMask.Paths, func(stored *ttnpb.ApplicationActivationSettings) (*ttnpb.ApplicationActivationSettings, []string, error) {
-		if stored == nil {
-			if err := ttnpb.RequireFields(req.FieldMask.Paths,
-				"kek_label",
-			); err != nil {
-				return nil, nil, errInvalidFieldMask.WithCause(err)
-			}
-			if !ttnpb.HasAnyField(req.FieldMask.Paths, "kek.key") ||
-				!ttnpb.HasAnyField(req.FieldMask.Paths, "kek.encrypted_key") && !ttnpb.HasAnyField(req.FieldMask.Paths, "kek.kek_label") {
-				return nil, nil, errInvalidFieldMask.WithCause(ttnpb.RequireFields(req.FieldMask.Paths, "kek.key"))
-			}
-		}
 		return &req.ApplicationActivationSettings, sets, nil
 	})
-	if errors.IsNotFound(err) {
```

Comment:
Without this, users get direct error from the (probably Redis) registry, it's nicer to wrap it, why change this?